From 4a54997e63577ce3161b8f86f729fe3951d730be Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20=C4=8Cerm=C3=A1k?= <dcermak@suse.com>
Date: Tue, 14 Jan 2020 14:20:40 +0100
Subject: [PATCH 2/8] Don't install pandoc and mathjax

Because we use the system wide version, there's no need to waste time installing
the files.
---
 src/cpp/session/CMakeLists.txt | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/src/cpp/session/CMakeLists.txt b/src/cpp/session/CMakeLists.txt
index d0b41b13d2..69d6cb2b2b 100644
--- a/src/cpp/session/CMakeLists.txt
+++ b/src/cpp/session/CMakeLists.txt
@@ -534,10 +534,6 @@ if (NOT RSTUDIO_SESSION_WIN32)
    install(DIRECTORY "${RSTUDIO_DEPENDENCIES_DIR}/common/dictionaries"
            DESTINATION "${RSTUDIO_INSTALL_SUPPORTING}/resources")
 
-   # install mathjax for local html preview
-   install(DIRECTORY "${MATHJAX_PATH}"
-           DESTINATION "${RSTUDIO_INSTALL_SUPPORTING}/resources")
-
    # icons for database connections
    install(DIRECTORY "resources/connections"
            DESTINATION ${RSTUDIO_INSTALL_SUPPORTING}/resources)
@@ -550,14 +546,6 @@ if (NOT RSTUDIO_SESSION_WIN32)
    install(DIRECTORY "resources/dependencies"
            DESTINATION ${RSTUDIO_INSTALL_SUPPORTING}/resources)
 
-   # install pandoc
-   set(PANDOC_BIN "${RSTUDIO_DEPENDENCIES_DIR}/common/pandoc/${PANDOC_VERSION}")
-   file(GLOB PANDOC_FILES "${PANDOC_BIN}/pandoc*")
-   set(PANDOC_FILES ${PANDOC_CITEPROC_BIN_LOCATION} ${PANDOC_BIN_LOCATION})
-   install(FILES ${PANDOC_FILES}
-           PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
-           DESTINATION  ${RSTUDIO_INSTALL_BIN}/pandoc)
-
    # install embedded packages
    foreach(PKG ${RSTUDIO_EMBEDDED_PACKAGES})
       file(GLOB PKG_FILES "${RSTUDIO_DEPENDENCIES_DIR}/common/${PKG}*.tar.gz")
-- 
2.30.2
